client/src/components/common/Logo.tsx

```diff
@@ -1,9 +1,12 @@
 import React from "react";
 import { css } from "@emotion/css";
+import { useParams } from "react-router-dom";
 
-export const Logo: React.FC<{ noWordmark?: boolean }> = ({
+export const Logo: React.FC<{ noWordmark?: boolean; artistId: boolean }> = ({
   noWordmark = false,
+  artistId: boolean,
 }) => {
+  const { artistId } = useParams();
```

Comment:
This doesn't cause an error defining the same variable twice? I think in settings like this it's useful to name booleans with a `is` prefix. Like `isArtist`. It makes it much clearer what the variable is containing. Right now it looks like the variable contains an artist id.

Reply:
Ah yeah, sorry I put that in because it was the proposed fix I think! I actually didn't even need to put, it worked fine without it, but it threw an error that's why I added it in.